src/festim/exports/surface_temperature.py

```diff
@@ -0,0 +1,102 @@
+import csv
+from dolfinx import fem
+import ufl
+import festim as F
+
+class SurfaceTemperature:
+    """Exports the average temperature on a given surface.
+
+    Args:
+        temperature_field (fem.Constant or fem.Function): the temperature field to be computed
+        surface (int or festim.SurfaceSubdomain): the surface subdomain
+        filename (str, optional): name of the file to which the average surface temperature is exported
+
+    Attributes:
+        temperature_field (fem.Constant or fem.Function): the temperature field
+        surface (int or festim.SurfaceSubdomain): the surface subdomain
+        filename (str): name of the file to which the surface temperature is exported
+        t (list): list of time values
+        data (list): list of average temperature values on the surface
+    """
+
+    def __init__(self, temperature_field, surface, filename: str = None) -> None:
+        self.temperature_field = temperature_field
+        self.surface = surface
+        self.filename = filename
+
+        self.t = []
+        self.data = []
+        self._first_time_export = True
+
+    @property
+    def filename(self):
+        return self._filename
+
+    @filename.setter
+    def filename(self, value):
+        if value is None:
+            self._filename = None
+        elif not isinstance(value, str):
+            raise TypeError("filename must be of type str")
+        elif not value.endswith(".csv") and not value.endswith(".txt"):
+            raise ValueError("filename must end with .csv or .txt")
+        self._filename = value
+
+    @property
+    def surface(self):
+        return self._surface
+
+    @surface.setter
+    def surface(self, value):
+        if not isinstance(value, (int, F.SurfaceSubdomain)) or isinstance(value, bool):
+            raise TypeError("surface should be an int or F.SurfaceSubdomain")
+        self._surface = value
```

Comment:
This is duplicated from `SurfaceQuantity`. Can be removed when this class inherits from it